src/components/Menu/index.vue

```diff
@@ -2,13 +2,13 @@
   <h1 id="menuItems">{{ greeting }}</h1>
   <div class="menu-filters">
     <button
-      v-for="name in menuProducts"
-      :key="name.id"
+      v-for="item in menuProductsCategory"
```

Comment:
Renaming menuProductsCategory to categories and changing "item in menuProductsCategory" to "category in categories" would make this v-for more clear